cht-petals/download.py

```diff
@@ -6,19 +6,16 @@
 from transformers import AutoTokenizer, LlamaTokenizer
 
 parser = argparse.ArgumentParser()
-parser.add_argument("--model", help="Model to download")
-parser.add_argument("--dht-prefix", help="DHT prefix to use")
```

Comment:
sorry, a bit confused about how it will be a breaking change.. as currently we are not using `dht-prefix` anywhere in docker builds and `download.py` is only being used for docker builds, but in other files `dht-prefix` is still being used.

Reply:
oh it's not used in the registry/app anywhere? Ok no problem dropping it then :)